packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -230,7 +230,9 @@ export type CreateChildSummarizerNodeFn = (summarizeInternal: SummarizeInternalF
 export interface IFluidDataStoreContext extends EventEmitter, Partial<IProvideFluidDataStoreRegistry> {
     readonly documentId: string;
     readonly id: string;
-    // True if the data store is LocalDataStoreContext.
+    // A data store created by a client, is a local data store for that client. Also, when a detached container loads
```

Comment:
Might be worth adding couple clarifications:
1. Does it stay true after local store is attached? Or it is reset to false?
2. This is good description in terms of what it means, but is there any message we want to send to runtime developers in terms of what they need to do with it? I.e. something in the format of "as implementer of data store runtime, you need to use this property to implement X differently"

Also some typo "doesn't exists on storage" -> "doesn't exists in storage"

Reply:
Fixed. About the message, I have added a very general message because I feel there could be any scenario in mind of developer using this property.